Update-731: DD and Scaling v3 docs

ntner opened the pull request:
### What is the feature/fix?

Overhauled our Datadog docs:
Removed our partial / outdated installation instructions and redirected everything to Datadog docs themselves.  We don't need to constantly update or support the copy/paste of their docs to our doc site. 

Added Scaling docs:
Datadog custom scaling instructions and example to the Scaling page.

### Does it has a breaking change?

no,  the guide/docs are working now

### How to use/test it?

Read the docs and try it!


### Checklist
- [ ] New coverage tests
- [ ] Unit tests passing
- [ ] E2E tests passing
- [ ] E2E downgrade/update test passing
- [ ] Documentation updated
- [ ] No warnings or errors on Deepsource/Codecov

heronrs reviewed (dismissed):
Nice one. Thanks

Twsouza reviewed (dismissed):
shouldn't it be specified from which version it is available?

ntner commented:
Good point @Twsouza, I found a couple more little typos as well.  Once Nahid gives it a look I will correct these things and push again.

nightfury1204 commented on docs/deployment/scaling.md:
```diff
@@ -67,3 +67,126 @@ You must consider that the targets for CPU and Memory use the service replicas l
+Deploy the service with `kubectl apply -f dd-agent-service.yaml`
+
+
+In this example we will use a simple [nodejs](https://github.com/convox-examples/nodejs) app to demonstrate how to setup scaling with this method. 
```
@ntner to push the metrics to datadog i made some changes in the code, https://github.com/convox-examples/nodejs doesn't have that.
Should I push the code in the different repository?
cc: @heronrs

ntner replied:
@nightfury1204 thanks for catching that... I will look at your example again and can make a new example repo with it.

ntner commented:
Updated with:
new repo and verbiage for the configured nodejs app / unconfigured example
added info regarding  regarding version requirement
updated rack update info to reflect how v3 racks should be updated.

nightfury1204 commented on docs/deployment/scaling.md:
```diff
@@ -67,3 +67,131 @@ You must consider that the targets for CPU and Memory use the service replicas l
+Deploy the service with `kubectl apply -f dd-agent-service.yaml`
+
+
+In this example we will start with a simple [nodejs app](https://github.com/convox-examples/nodejs) to demonstrate how to setup scaling with this method. 
```
`https://github.com/convox-examples/nodejs` to `https://github.com/convox-examples/nodejs-dd-autoscale`?

nightfury1204 commented on docs/deployment/scaling.md:
````diff
@@ -67,3 +67,131 @@ You must consider that the targets for CPU and Memory use the service replicas l
+``` 
+[Configured Nodejs app](https://github.com/convox-examples/nodejs-dd-autoscale)
+
+You can now deploy the application with `convox apps create` then `convox deploy`
````
`convox apps create -a nodejs` then `convox deploy -a nodejs`
since by default takes the directory name and this has `https://github.com/convox-examples/nodejs-dd-autoscale` different name.
or you can change the `-a nodejs` on the below commands

nightfury1204 reviewed (approved):
:fire:
